src/integration.test.js

```diff
@@ -1,29 +1,97 @@
-/* global it describe beforeEach expect */
+/* global jest it describe expect */
 
 import React from 'react'
-import Enzyme, { mount, shallow } from 'enzyme'
+import Enzyme, { shallow } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
-import App from './App'
+import PasteInlineForm from './pasteInlineForm'
+import PasteBox from './pasteBox'
+import fetch from 'jest-fetch-mock'
 
 Enzyme.configure({adapter: new Adapter()})
+window.fetch = fetch
 
 describe('integration testing', () => {
   // main integration testing
+  it('renders the right text in the code box', () => {
+    const bumBumyData = {
+      codeText: 'this is code',
+      handleChange: jest.fn()
+    }
+    const appC = shallow(<PasteBox {...bumBumyData} />)
+    expect(appC.find('#pastingText').props()['value']).toBe('this is code')
+  })
+
+  it('renders the right text in the title box', () => {
+    // dummy data
```

Comment:
also useless comment? There is no corresponding comment on line 16, for example.